src/main/java/com/racingcar/game/car/Car.java

```diff
@@ -14,12 +16,12 @@ public Car(String name) {
         this.name = name;
     }
 
-    public void move(int value) {
-        if (value < MOVE_STANDARD) {
-            return;
+    public RoundCarRecord move(int value) {
+        if (value >= MOVE_STANDARD) {
+            distance++;
         }
 
-        distance++;
+        return RoundCarRecord.of(this);
```

Comment:
```suggestion
        return new RoundCarRecord(this.name, this.distance);
```
이 방식도 좋지만 요렇게하면 getter를 사용하지 않고 상태를 전달할 수 있어서 더 좋을 것 같습니다^_^